lib/type/wizard/store.js

```diff
@@ -5,6 +5,19 @@ var Util = require( 'findhit-util' );
 
 function Store ( route ) {
 
+    // If req doesn't have a session, this route type should NOT work, throw Error!
+    if( ! route.req[ route.options.reqSessionKey ] ) {
```

Comment:
Here it should be `route.router.options.reqSessionKey` instead, i'm currently testing this with our app.

I will make this change.